content/topics/intro-to-relational-databases/_index.md

```diff
@@ -1,15 +1,99 @@
+
 ---
-title: SQL
-ready: True
+title: Introduction to relational database
+ready: true
 ---
 
-## Tutorial
 
-Complete the DataCamp Intro and Intermediate SQL courses:
+### INTRODUCTION TO RELATIONAL DATABASE (SQL)
+
+
+#### What is Relational DataBase? 
+
+Learn more here:[What is RDBMS](https://www.youtube.com/watch?v=h8IWmmxIyS0)
+
+- Introduction to SQL relational database [visit](https://www.mikedane.com/databases/sql/)
+
+- Basic SQL queries: [See these](https://www.mikedane.com/databases/sql/basic-queries/)
+
+
+#### LESSONS
+
+##### Lesson1:
+
+- Keys / Data and Tables:
+> [check these out](https://www.mikedane.com/databases/sql/tables-keys/)
+
+##### Lesson2:
+
+- Basics/Elements of SQL 
+> [check these out](https://www.mikedane.com/databases/sql/sql-basics/)
+
+##### Lesson3:
+  
+- SQL-Installation:
+   
+    > [windows](https://www.mikedane.com/databases/sql/mysql-windows-installation/)
+
+    > [MAC / Linux/ Ubuntu](https://www.mikedane.com/databases/sql/mysql-mac-installation/)
+
+##### Lesson4:
+  
+- Creating Tables
+> [Click here](https://www.mikedane.com/databases/sql/creating-tables/)
+
+##### Lesson5:
+  
+- Inserting Data
+> [check it out](https://www.mikedane.com/databases/sql/inserting-data/)
+  
+**Lesson6:**
+  
+- Constraints
+> [check it out](https://www.mikedane.com/databases/sql/constraints/)  
+
+##### Lesson7:
+
+- Update & Delete Data
+> [check it out](https://www.mikedane.com/databases/sql/basic-queries/)
+
+##### Building APIs:
+
+- Nodejs API with SQL:
```

Comment:
This is out of place.

Node and sql is something seprate. We ususally get peeps to understand SQL by itself, then we tie it back to the language that they use later on.

Also, let's stay away from MySQL. Only Postgres. MySQL isn't actually fully sql compliant, Postgres is waaay more solid. So don't go telling people to use MySQL. 

This content is all Mysql, you can just remove this section completely